### ISSUE TITLE
Modal - wrapped "title" text does not line up with widget

### ISSUE THREAD

**Issue by Shando** [spaghett1c0de/DataTable-AltEditor#6]
Hi all,

I've just noticed that, in the modal pop-up, if the "title" for a columnDef wraps then it does not line up with the widget (see attached image).

Regards

Shando

![title not lining up](https://user-images.githubusercontent.com/758076/82955007-3f877c80-9ff1-11ea-91df-f54c64f6dde3.PNG)

**Comment by spaghett1c0de** [comment]
@Shando I noticed that too. But that appears to be how Bootstrap wraps the label. It automatically wraps and expands the label height when the text gets too long.
<img width="515" alt="Screen Shot 2020-05-29 at 2 57 13 PM" src="https://user-images.githubusercontent.com/18053491/83308900-b2911e80-a1bc-11ea-9215-778cd122a7ae.png">
I tested this with a jsfiddle and this is the result.

Best,
Vincent
